CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 * Maintenance
   * Switched pydineof_dineof load code to use newer `pysat.utils.io.load_netcdf`
   * Updated NEP29 and GitHub actions versions
+  * Added manual test for pysat and pysatNASA Release Candidates
```

Comment:
Because this is manual, some instructions for when/how it should be run would be useful.  Perhaps in the Contributing file?

Reply:
Updated the pysat wiki here: https://github.com/pysat/pysat/wiki/Checklist-for-Release

An example of what the run workflow button looks like is here: https://github.com/pysat/pysat/actions/workflows/stats.yml

Reply:
I updated your wiki instructions to make them easier to read and find.